modules/oe_corporate_blocks_demo/oe_corporate_blocks_demo.post_update.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate blocks Demo post updates.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\block\Entity\Block;
+
+/**
+ * Delete the block siteswitcherblock.
+ */
+function oe_corporate_blocks_demo_post_update_00001_delete_the_siteswitcherblock() {
```

Comment:
We should use the [new convention](https://github.com/openeuropa/documentation/blob/master/docs/architecture/third-party/drupal/module-update-policy.md) here.

Reply:
done